src/components/content-presentation/tabs/Tabs.tsx

```diff
@@ -2,7 +2,7 @@
 import classNames from 'classnames';
 import React, { FC, HTMLAttributes, useEffect } from 'react';
 import HeadingLevel, { HeadingLevelType } from '@util/HeadingLevel';
-import TabsJs from 'nhsuk-frontend/packages/components/tabs/tabs';
+import TabsJs from './../../../scripts/tabs';
```

Comment:
could we put these under an alias

Reply:
I did try this but rollup wasn't able to resolve the files when using an alias, I'll give it a fresh go and see if I've missed anything